iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/LogicalPlanVisitor.java

```diff
@@ -522,6 +527,9 @@ public PlanNode visitShowTimeSeries(
               .planTimeseriesRegionScan(analysis.getDeviceToTimeseriesSchemas(), false)
               .planLimit(limit)
               .planOffset(offset);
+      if (showTimeSeriesStatement.hasOrderByComponent()) {
+        planBuilder.planOrderBy(showTimeSeriesStatement.getOrderByComponent().getSortItemList());
+      }
```

Comment:
Same, should this executed after limit/offset?

Reply:
I will move OrderBy to be executed after limit/offset.